use-timescale/upgrades.md

```diff
@@ -71,14 +71,15 @@ For more information about replicas, see the
 
 ## Non-critical maintenance updates
 
-Non-critical upgrades are made available before the upgrade is performed
-automatically. During this time you can click `Apply upgrades` to start the
-upgrade at any time. However, after the time expires, usually around a week,
-the upgrade is triggered automatically in the next available maintenance window
-for your service. You can configure the maintenance window so that these
-upgrades are started only at a particular time, on a set day of the week. If
-there are no pending upgrades available during a regular maintenance window, no
-changes are performed.
+Non-critical upgrades are available before the upgrade is performed
+automatically by $CLOUD_LONG. To upgrade $TIMESCALE_DB manually, run `ALTER
+EXTENSION timescaledb UPDATE` in your $SERVICE_LONG, or `Pause` and `Resume`.
+This triggers the upgrade in the next available maintenance window. You can
```

Comment:
```suggestion
If no action is taken by the user, upgrade is triggered in the next available maintenance window. You can
```

Reply:
@billy-the-fish with all the suggested changes committed, the meaning I wanted to convey got distorted a bit. Could you have a look at this sentence?